code/modules/food_and_drinks/food/foods/meat.dm

```diff
@@ -47,6 +47,12 @@
 	desc = "Much meatier than monkey meat."
 	list_reagents = list("nutriment" = 5, "vitamin" = 1)
 
+/obj/item/food/snacks/meat/kangaroo
+	name = "kangaroo meat"
+	desc = "Extremely muscular and tender meat"
+	list_reagents = list("protein" = 4, "iron" = 5, "vitamin" = 1)
+	tastes = list("punch in the face" = 1, "fowl" = 3)
```

Comment:
```suggestion
	tastes = list("being punched in the face" = 1, "fowl" = 3)
```

Reply:
You can taste being punched in the face

Reply:
no, i prefer taste punch in the face, BEING as taste flavor is not what world need.

Reply:
This would come out as "you can taste punch in the face." If that's the case, this is grammatically incorrect.
It should be "you can taste a punch in the face," or "you can taste being punched in the face."